java/core/src/main/java/com/microsoft/bond/protocol/StructMetadata.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.bond.protocol;
+
+/**
+ * A metadata describing a struct, used by {@link ProtocolWriter} for certain protocol types.
+ */
+public interface StructMetadata {
```

Comment:
We will take another look at this when designing untagged protocols.

---
In reply to: [123385305](https://github.com/Microsoft/bond/pull/502#discussion_r123385305) [](ancestors = 123385305)